lib/impl/editing.dart

```diff
@@ -91,6 +88,19 @@ bool _handleEnterKey(TextEditor editor, int row, int col) {
     return true;
   }
 
+  if (trimmedText.startsWith('//')) {
```

Comment:
`if (trimmedText.startsWith('//') && !trimmedText.startsWith('///') {`? Or do you intend this to also work in dartdoc comments?